.github/workflows/Linux_options.yml

```diff
@@ -30,16 +30,27 @@ jobs:
         - {
             options: "-DUSE_IPOLATES=1"
           }
+        - {
+            options: "-DUSE_AEC=ON"
+          }
```

Comment:
At the very bottom of the workflow file, you can see that it's building wgrib2 with:

`make VERBOSE=1
`
But it is not running the tests.

After the make VERBOSE=1 line, add the following:

`  ctest --verbose --output-on-failure --rerun-failed
`
Then it will run your test.